### PR TITLE
Quickfix/adapt master python sdk

### PR DESCRIPTION
**Pull request by zhangchi1**
**What this PR does / why we need it**:
Adapt the latest python SDK in master branch
1. update client and file with DATA_FORMAT
2. update python SDK tests: tests/loaders/test_file.py
**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: sdk/python/feast/loaders/file.py [hunk 1]
```diff
@@ -13,21 +13,24 @@
...
+from google.cloud import storage
 from pandavro import to_avro
 
+from feast.serving.ServingService_pb2 import DataFormat
 from feast.staging.storage_client import get_staging_client
 
 
 def export_source_to_staging_location(
-    source: Union[pd.DataFrame, str], staging_location_uri: str
+        source: Union[pd.DataFrame, str], staging_location_uri: str, data_format: DataFormat
```
Comment (dr3s) [inline sdk/python/feast/loaders/file.py]: indent issue

File: sdk/python/feast/loaders/file.py [hunk 2]
```diff
@@ -95,7 +99,7 @@ def export_source_to_staging_location(
     # Push data to required staging location
     get_staging_client(uri.scheme).upload_file(
         source_path, uri.hostname, str(uri.path).strip("/") + "/" + file_name,
-    )
+                                   )
```
Comment (dr3s) [inline sdk/python/feast/loaders/file.py]: indent fix

File: sdk/python/feast/loaders/file.py [hunk 3]
```diff
@@ -51,8 +54,9 @@ def export_source_to_staging_location(
             Remote staging location where DataFrame should be written.
             Examples:
                 * gs://bucket/path/
-                * s3://bucket/path/
```
Comment (jmelinav) [inline sdk/python/feast/loaders/file.py:54]: Undo this change.

**Review by dr3s — APPROVED** [review]
